scripts/pi-hole/php/sub.php

```diff
@@ -27,13 +27,16 @@
 
 switch($type) {
     case "white":
-        exec("sudo pihole -w -q -d ".$domain);
+        echo shell_exec("sudo pihole -w -q -d ".$domain);
         break;
     case "black":
-        exec("sudo pihole -b -q -d ".$domain);
+        echo shell_exec("sudo pihole -b -q -d ".$domain);
```

Comment:
What is the reason for this change?

Reply:
This will make the `sub.php` page pass through warnings and errors. `add.php` does this as well.